tests/performance/apdex-score/sample-settings-files/default-config-navigation-test.json

```diff
@@ -0,0 +1,154 @@
+{
+  "iterations": 1,
+  "instanceURL": "<instance url>",
+  "hasPrivacyPolicy": false,
+  "capabilities": [
+    {
+      "platformVersion": "<android version>",
+      "deviceName": "<phone model>",
+      "appPath": "<path to cht android apk>",
+      "noReset": false
+    }
+  ],
+
+  "skip": {
+    "login": false,
+
+    "loadContactList": false,
+    "loadCHWArea": false,
+    "loadHousehold": false,
+    "loadPatient": false,
+    "loadMessageList": false,
+    "loadTaskList": false,
+    "loadAnalytics": false,
+    "loadReportList": false,
+    "searchContact": false,
+    "searchReport": false,
+
+    "submitTask": true,
+    "createPatient": true,
+    "submitPatientReport": true
+  },
+
+  "users": [
+    {
+      "type": "offline",
+      "role": "chw",
+      "username": "<username>",
+      "password": "<password>"
+    }
+  ],
+
+  "pages": {
+    "contactList": {
+      "navigation": [
+        { "selector": "//*[@text=\"Reports\"]" },
+        {
+          "selector": "//*[@text=\"People\"]",
+          "asserts": [ { "selector": "//*[contains(@text, \"North Karsoncester\")]" } ]
+        }
+      ],
+      "search": {
+        "value": "Latoya Waelchi",
+        "asserts": [
+          { "selector": "//*[@text=\"Latoya Waelchi\"]" },
+          {
+            "scrollDown": 5,
+            "selector": "//*[contains(@text, \"No more people\")]"
+          }
+        ],
+        "postTestPath": [
+          { "selector": "//*[@text=\"Reports\"]" }
```

Comment:
Is this so we unload the page? I wonder if we can just make this as a generic unload instead of having to specify here. Like always go to the about page or something.

Reply:
The test design may not allow this - as tests are designed to continue from where the previous test stopped